### PR TITLE
implement PVRT support decompress texture.

### PR DESCRIPTION
**Pull request by Zangetsu38**
- implement PVRT texture support 
- [x] PVRT-I full support
- [x] PVRT-II good support

# Result: 
- Sackboy runner:
image

- Fort defense
image

- Salt and Sanctuary
image
image
image
blocked here, touch screen issue ? 
image

- Silent hill
image
image
image
image

### CODE REVIEW

**Comment by Zangetsu38** [comment]
is ready for review/merge

File: vita3k/renderer/include/renderer/pvrt-dec.h [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+* THE SOFTWARE.
+*/
+
+/*!
+\brief Contains functions to decompress PVRTC or ETC formats into RGBA8888.
+\file PVRCore/texture/PVRTDecompress.h
+\author PowerVR by Imagination, Developer Technology Team
+\copyright Copyright (c) Imagination Technologies Limited.
+*/
```
Comment (1whatleytay) [inline vita3k/renderer/include/renderer/pvrt-dec.h:32]: Triple checking we are supposed to have access to this...

File: vita3k/renderer/src/gl/texture.cpp [hunk 2]
```diff
@@ -106,6 +108,12 @@ static size_t decompress_compressed_swizz_texture(SceGxmTextureBaseFormat fmt, v
         renderer::texture::decompress_bc_swizz_image(width, height, reinterpret_cast<const std::uint8_t *>(data),
             reinterpret_cast<std::uint32_t *>(dest), ubc_type);
         return (((width + 3) / 4) * ((height + 3) / 4) * ((ubc_type > 1) ? 16 : 8));
+    } else if ((fmt >= SCE_GXM_TEXTURE_BASE_FORMAT_PVRT2BPP) && (fmt <= SCE_GXM_TEXTURE_BASE_FORMAT_PVRTII4BPP)) {
+        // TODO, is not perfect for PVRT-II.
+        pvr::PVRTDecompressPVRTC(data, is_2pp, width, height, (fmt == SCE_GXM_TEXTURE_BASE_FORMAT_PVRTII2BPP) || (fmt == SCE_GXM_TEXTURE_BASE_FORMAT_PVRTII4BPP),
+            reinterpret_cast<uint8_t *>(dest));
+        // TODO, calcule return maybe not right.
```
Comment (1whatleytay) [inline vita3k/renderer/src/gl/texture.cpp:115]: lgtm, why do you think so?

File: vita3k/renderer/src/gl/texture.cpp [hunk 3]
```diff
@@ -84,6 +85,7 @@ void configure_bound_texture(const SceGxmTexture &gxm_texture) {
  */
 static size_t decompress_compressed_swizz_texture(SceGxmTextureBaseFormat fmt, void *dest, const void *data, const std::uint32_t width, const std::uint32_t height) {
     int ubc_type = 0;
+    const int is_2pp = (fmt == SCE_GXM_TEXTURE_BASE_FORMAT_PVRT2BPP) || (fmt == SCE_GXM_TEXTURE_BASE_FORMAT_PVRTII2BPP);
```
Comment (1whatleytay) [inline vita3k/renderer/src/gl/texture.cpp:88]: 2**b**pp right?

File: vita3k/renderer/src/gl/texture.cpp (hunk 2 above)
Comment (Zangetsu38) [reply vita3k/renderer/src/gl/texture.cpp:115]: i have tested few value and get same for all so not sur ^^

File: vita3k/renderer/src/gl/texture.cpp (hunk 3 above)
Comment (Zangetsu38) [reply vita3k/renderer/src/gl/texture.cpp:88]: ? 2bpp is 2pp ^^, is not right, if is 2bpp is send 1 and if is not send 0

**Pull request merged**